### DISCUSSION TITLE
Changing private profile back to public

### DISCUSSION THREAD

**Discussion by geeky01adarsh** [community/community#80402] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey community!

I just got to know the amazing features of making your Github account private. 
However, I was confused about what would happen if I changed to make my profile back to public. Will my contributions during the time span of profile being private would be counted or not?

I want to show them after maybe 2-3 months, if not on there feed then on my contribution graph and stats. Is it possible?

**Comment by jane-does-coding** [comment]
You can choose if you want your contributions in private repositories to be shown or not. On your profile page where it shows the yearly contributions chart on the top right part of it there is a label "Contribution settings" click on it and you can choose if you want private contributions to be counted or not

**Comment by TriSBie** [comment]
Great

**Reply by geeky01adarsh** [reply]
@TriSBie ?

**Comment by Nargeswww** [comment]
Hello

**Reply by geeky01adarsh** [reply]
Hi @Nargeswww!!!
Do you have anything to share?

**Reply by leilawww** [reply]
hello

**Comment by Dheeraj-02NK** [comment]
Keeping a repo private when its in Progress and then making it public once the project is finish is a good practice

**Comment by icarus** [comment, marked as answer]
Hey @geeky01adarsh. Hope you are doing well.

When you switch your GitHub profile from private back to public, contributions made during the private period won't automatically show up in your public contribution graph or stats. This is because GitHub respects the privacy settings active when those contributions were made.

However, if you want these contributions to be visible after your profile is public again, you’ll need to ensure the repositories where you made contributions during the private period are also set to public. Once both your profile and the repositories are public, those contributions should appear on your contribution graph and stats.

Just a heads up, turning a private repository public will make all its contents, including past contributions, visible to everyone. So, be mindful of the data in those repositories before changing their visibility.

Hope this helps! 💫

**Reply by geeky01adarsh** [reply]
Hey @icarus,
This was exactly what I was looking for. Thanks a lot for this amazing explanation.

**Comment by leilawww** [comment]
hello your muy fti jon

**Reply by Nargeswww** [reply]
Wooooooooooooooo hellloooo lila
